src/modules/common/components/Tile.css

```diff
@@ -28,6 +32,7 @@
 .thumbnail img {
   position: absolute;
   top: 0;
+  left: 0;
```

Comment:
Oh shit I just realized that I shouldn't do that... Or at least add a ' right: 0;' as well!

Reply:
It would be nice to have a linting rule preventing stuff like this.

Reply:
Yeah sure, but a linting rule demanding 'right' if you've used 'left' and the other way round should be okay. Or at least displaying a warning.f

Reply:
Yeah, but I'd guess we'd have to implement such a linting rule ourselves :D
(Not sure though, if it really doesn't exist :see_no_evil:)

Reply:
Yep, with a quick search I couldn't find anything suitable. Should we create a task for this?

Reply:
If you want you can create one, but I'm not really optimistic that that would be an easy task 😅

Reply:
If you want you can create one, but I'm not really optimistic that that would be an easy task 😅